modules/bibconvert/etc/authorlist2marcxml.xsl

```diff
@@ -88,7 +88,7 @@ along with Invenio; if not, write to the Free Software Foundation, Inc.,
   <xsl:template name="print-h-authorid">
```

Comment:
I know the name is now wrong, but I don't know XSLT well enough to confidently merge this with the `print-j-authorid` rule.

Reply:
It's just a function name. Wherelse is it used? You just need to update it there aswell.

Reply:
yeah, I was overthinking this.

Reply:
But can you then amend it everywhere to be `print-j-authorid`?

Reply:
done